utils/images.ahk

```diff
@@ -27,3 +27,21 @@ GetCurrentMapPage() {
         }
     }
 }
+
+SearchUpgrade1() {
+	if (SearchImage("buttons\upgrades\1R", "*TransBlack", 250, 475, 1494, 499) or SearchImage("buttons\upgrades\1L", "*TransBlack", 250, 475, 1494, 499)) {
+    return true
+}
+}
+
+SearchUpgrade2() {
+	if (SearchImage("buttons\upgrades\2R", "*TransBlack", 250, 625, 1494, 649) or SearchImage("buttons\upgrades\2L", "*TransBlack", 250, 625, 1494, 649)) {
+    return true
+}
+}
+
+SearchUpgrade3() {
+	if (SearchImage("buttons\upgrades\3R", "*TransBlack", 250, 775, 1494, 799) or SearchImage("buttons\upgrades\3L", "*TransBlack", 250, 775, 1494, 799)) {
+    return true
+}
+}
```

Comment:
These should be just one function instead of three

Reply:
Not sure how to put these into one function.

Reply:
done